src/core/Tooltip/index.tsx

```diff
@@ -35,7 +35,9 @@ const Tooltip = (props: TooltipProps): JSX.Element => {
     props,
   });
 
-  return <RawTooltip {...rest} arrow classes={{ arrow, tooltip }} />;
+  return (
+    <RawTooltip arrow classes={{ arrow, tooltip }} interactive {...rest} />
```

Comment:
`interactive` makes it so that users can hover on the tooltip itself and interact with its content (e.g. clicking on a `Learn More` link)